src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -292,11 +288,11 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
                     </TableHeadingCell>
                     <TableHeadingCell scope="col">
                       <TextInput
-                        id="filterByRoomName"
-                        name="filterByRoomName"
+                        id="room-filter"
+                        name="room-filter"
                         placeholder="Filter by Room Name"
                         value={roomFilter}
-                        label="The table will be filtered by selected room name"
+                        label="Change to filter the list of meetings by room"
```

Comment:
The room admin is not related to meetings, so this label text should be updated as well.